src/BenchmarksApps/Kestrel/PlatformBenchmarks/Data/BatchUpdateString.cs

```diff
@@ -11,8 +11,8 @@ internal class BatchUpdateString
 
         public static DatabaseServer DatabaseServer;
 
-        internal static readonly string[] Ids = Enumerable.Range(0, MaxBatch).Select(i => $"@Id_{i}").ToArray();
-        internal static readonly string[] Randoms = Enumerable.Range(0, MaxBatch).Select(i => $"@Random_{i}").ToArray();
+        internal static readonly string[] Ids = Enumerable.Range(0, MaxBatch).Select(i => $"@I{i}").ToArray();
```

Comment:
He's at it again :rofl:

Reply:
More seriously, when we finally do raw SQL in Npgsql (https://github.com/npgsql/npgsql/issues/3200#issuecomment-708936165) we will start using native PostgreSQL placeholders, which are positional, so we'll have $1, $2.